src/sentry/issues/endpoints/group_similar_issues_embeddings.py

```diff
@@ -41,24 +41,26 @@ class GroupSimilarIssuesEmbeddingsEndpoint(GroupEndpoint):
         "GET": ApiPublishStatus.PRIVATE,
     }
 
-    def get_group_hashes_for_group_id(self, group_id: int) -> set[str]:
-        hashes = GroupHash.objects.filter(group_id=group_id)
-        return {hash.hash for hash in hashes}
+    def get_group_id_from_hash(self, hash: str, project_id: int) -> int:
+        group_hash = GroupHash.objects.get(project_id=project_id, hash=hash)
+        return group_hash.group_id
```

Comment:
as written, this will cause N database queries as we're doing a `get` in a loop.

can we instead get the list of group_ids with a filter query outside the loop, and use the results inside the loop?

Reply:
yeah good callout - thanks! it'll just be 10 since that's how many neighbors we return, but this is definitely better